plugins/CuraEngineBackend/StartSliceJob.py

```diff
@@ -269,9 +278,15 @@ def isCancelled(self):
     #   \return A dictionary of replacement tokens to the values they should be
     #   replaced with.
     def _buildReplacementTokens(self, stack) -> dict:
+        default_extruder_position = Application.getInstance().getMachineManager().defaultExtruderPosition
```

Comment:
you do need it here because you use it in one place. 
value = Application.getInstance().getMachineManager().defaultExtruderPosition